lsp-mode.el

```diff
@@ -2580,7 +2580,14 @@ an Elisp regexp."
 (define-minor-mode lsp-mode ""
   nil nil nil
   :keymap lsp-mode-map
-  :lighter (:eval (lsp-mode-line))
+  :lighter
+  '(" LSP["
+    (lsp--cur-workspace
+     (:eval (lsp--workspace-print lsp--cur-workspace))
```

Comment:
there is no need to handle `lsp--cur-workspace`, that leg can be removed.

Reply:
Done